pages/cas/rating.vue

```diff
@@ -0,0 +1,146 @@
+  return $fetch('/api/club/rating/available', {
+    headers: useRequestHeaders(),
+    method: 'GET',
+  })
+})
+
+if (!data.value) {
+  throw createError({
+    statusCode: 500,
+    message: '服务器错误',
+  })
+}
+
+const { handleSubmit, resetForm } = useForm({
+  validationSchema: formSchema,
+})
+
+function onSubmitRating() {
+  return handleSubmit(async (values) => {
+    isLoading.value = true
+    const ratingData = {
+      club: values.club,
+      date: new Date().toISOString(),
+      score: values.score,
+      comment: values.comment || 'null',
+    }
+    const { error } = await useFetch('/api/club/rating/new', {
+      headers: useRequestHeaders(),
+      method: 'post',
+      server: false,
+      body: ratingData,
+    })
+    if (error.value) {
+      toast({
+        title: '错误',
+        description: '请稍后再试',
+        variant: 'destructive',
+      })
+    }
+    isLoading.value = false
+    emit('refresh')
+    resetForm()
+  })
+}
+</script>
+
+<template>
+  <div class="w-full">
+    <Card class="w-full">
+      <CardHeader>
+        <CardTitle class="flex items-center gap-x-1">
+          <Icon name="material-symbols:add-circle-outline" />
+          新增
+        </CardTitle>
+        <CardDescription>在此处加入社团进行评分</CardDescription>
+      </CardHeader>
+      <CardContent>
+        <form class="space-y-6" @submit="onSubmitRating">
+          <FormField v-slot="{ componentField, value }" name="club">
+            <FormItem>
+              <FormLabel>社团</FormLabel>
+              <Select v-bind="componentField">
+                <FormControl>
+                  <SelectTrigger :class="cn('w-full ps-3 text-start font-normal hover:bg-muted', !value && 'text-muted-foreground')" variant="outline" :disabled="isLoading">
+                    <SelectValue placeholder="选择您需要評分的社团..." />
+                  </SelectTrigger>
+                </FormControl>
+                <SelectContent>
+                  <SelectGroup v-if="data">
+                    <SelectItem v-for="club in data" :key="club.id" :value="String(club.id)">
+                      {{ club.name.zh }}
+                    </SelectItem>
+                  </SelectGroup>
+                </SelectContent>
+              </Select>
+              <FormMessage />
+            </FormItem>
+          </FormField>
+          <FormField v-slot="{ componentField, value }" name="score">
+            <FormItem class="flex flex-col">
+              <FormLabel>社团评分：</FormLabel>
+              <Select v-bind="componentField">
+                <SelectTrigger :class="cn('w-full ps-3 text-start font-normal hover:bg-muted', !value && 'text-muted-foreground')" variant="outline" :disabled="isLoading">
+                  <SelectValue placeholder="选取该社团的评分" />
+                </SelectTrigger>
+                <SelectContent>
+                  <SelectGroup>
+                    <SelectLabel>Scores</SelectLabel>
+                    <SelectItem v-for="n in 5" :key="String(n)" :value=n>
```

Comment:
do as what the linter says

Reply:
The value n is a number, thus should not be enclosed in ".

Reply:
For vue.js and other similar web frameworks, the `"`s here on an element is used to wrap a javascript expression. Just like how you did in `v-for="n in 5"`.